rllib/examples/inference_and_serving/policy_inference_after_training_with_dt.py

```diff
@@ -0,0 +1,166 @@
+"""
+Example showing how you can use your trained Decision Transformer (DT) policy for
+inference (computing actions) in an environment.
+"""
+import argparse
+import gym
+import os
+
+import ray
+from ray import air, tune
+from ray.rllib.algorithms.dt import DTConfig
+from ray.rllib.algorithms.registry import get_algorithm_class
+from ray.tune.utils.log import Verbosity
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--num-cpus", type=int, default=0)
```

Comment:
make default None so that this won't become a problem if ray is already initialized (e.g. on a cluster)

Reply:
tricky: I just realized `foo = 0 or None` makes `foo` None

Reply:
yeah i took this from the other examples in the folder